packages/react-components/src/components/chart/hooks/useTrendCursors.ts

```diff
@@ -0,0 +1,35 @@
+import { UseTrendCursorsProps } from '../types';
+import useDataStore from '../../../store';
+import useResize from './useResize';
+import useSync from './useSync';
+import useEvents from './useEvents';
+
+const useTrendCursors = ({
```

Comment:
Why don't return the trend cursor data to the consumer of the hook?

Reply:
we dont need to, given we will be updating it using setGraphic, this is also because initially the callback were part of graphic options, which now is moved to be events on the chart instead. 
I could return it instead of updating the state, not entirely sure if one is better than other.